### ISSUE TITLE
[ISSUE] Thread pool starvation

### ISSUE THREAD

**Issue by rohithmanivarma123** [openbullet/OpenBullet2#234]
![IMG_20210420_105640_899](https://user-images.githubusercontent.com/32290473/115342139-66c9d800-a1c7-11eb-87f9-4ca4a804b2d4.jpg)
After using openbullet 2 in AWS vps after 1 or 2 hours it crashes and when I open vps it shows like this. Please fix this

**Comment by openbullet** [comment]
Please open the `appsettings.json` file in the base folder of OB2 and reduce the number of workers to like 200 or 400 and restart OB2. Let me know if it helps!

**Comment by rohithmanivarma123** [comment]
I'm using this in AWS vps (Linux) hence I can't open it. And I'm using 25 workers..

**Comment by openbullet** [comment]
I see you're using this https://github.com/certix7/openbullet2-installer, it should put your OB2 files inside the `/opt/openbullet2/` folder, so just do

```bash
cd /opt/openbullet2/
nano appsettings.json
```
Change the workers from 1000 to 200 or 400 and then CTRL+O CTRL+X to close it, then relaunch it.

**Comment by openbullet** [comment]
Please let me know if you managed to fix the problem so I can close the issue.

**Comment by rohithmanivarma123** [comment]
actually what I'm doing is I created 2 screens in ubuntu with a screen command. and I am using open bullet in one screen and ngrok in another screen.  I did what you said on the main screen. here are the screenshots and I didn't see anything like workers. and I request you to create a telegram/discord channel to share the problems of the open bullet. And this is also a request on how to use open bullet in ubuntu. and that repo https://github.com/certix7/openbullet2-installer isn't updating. I request you to create a repo on how to install it in Linux
![Screenshot 2021-04-21 015751](https://user-images.githubusercontent.com/32290473/115485991-5e2edb80-a273-11eb-8246-b9fa2d8d2e52.png)
![Screenshot 2021-04-21 015816](https://user-images.githubusercontent.com/32290473/115485992-5ec77200-a273-11eb-8851-c94a523013df.png)
